mods/d2k/rules/infantry.yaml

```diff
@@ -49,6 +42,7 @@ engineer:
 	Captures:
 		CaptureTypes: husk
 	-AutoTarget:
+	-TakeCover:
```

Comment:
I have an even better idea, I'll take a look at the asset browser and just add those sequences ;)